src/arctic3d/modules/pdb.py

```diff
@@ -676,26 +608,38 @@ def validate_api_hit(
             check_list.append(True)
         else:
             check_list.append(False)
+            reason = "coverage"
         # check resolution value
         if resolution is None:
             if "NMR" in exp_method:
                 check_list.append(True)
             else:
                 check_list.append(False)
+                reason = "None resolution"
         elif resolution < resolution_cutoff:
             check_list.append(True)
         else:
             check_list.append(False)
+            reason = "resolution"
+
+        # check chain ID not longer than 1 character
+        if len(chain_id) == 1:
+            check_list.append(True)
+        else:
+            check_list.append(False)
+            reason = "chain ID too big"
```

Comment:
Can it actually be more than 1 char? according to the format it should not - https://mmcif.wwpdb.org/dictionaries/mmcif_pdbx_v40.dic/Items/_atom_site.label_asym_id.html or is this just a check for user inputted data?